spec/support/factories.rb

```diff
@@ -0,0 +1,18 @@
+FactoryGirl.define do
+  factory :user do
+    sequence(:username) {|n| "relm #{n}"}
+    sequence(:first_name) { |n| "first#{n}"}
+    sequence(:last_name) {|n| "last#{n}"}
+    admin false
+    sequence(:email) {|n| "hello#{n}@world.com"}
+    sequence(:profile_url) {|n| "https://www.pic#{n}.com"}
+    sequence(:password) {|n| "Seekrit#{n}#{n}#{n}#{n}"}
+  end
+
+  factory :park do
+    sequence(:title)  {|n| "park#{n}"}
```

Comment:
Unnecessary spacing detected.<br>Space between { and | missing.<br>Space missing inside }.